towncrier.toml

```diff
@@ -1,7 +1,7 @@
 [tool.towncrier]
-version = "2.16.0"
+version = "2.15.1"
 directory = "doc/whatsnew/fragments"
-filename = "doc/whatsnew/2/2.16/index.rst"
+filename = "doc/whatsnew/2/2.15/index.rst"
```

Comment:
This is going to NOT be a merge conflict and we need to be careful when merging in main.

Reply:
I was wondering about this. Do we need to bump to the dev version on `main`? What's the benefit? Can't we just bump when we need to for a release?

Reply:
Well it's useful to know the pylint version we're talking about (2.16.0-dev0 vs 2.15.1). scm_setuptool was nice for this but created a dependencie we did not want...

Reply:
Yeah I mean I don't really mind, but referring to `main` would also work for me.
It reduces the amount of work necessary for a release so I thought it might be something to consider.

Reply:
referring to main ?

Reply:
As in, I never use `2.16.0-dev0` directly in any of my messages. I often just say `main` or something like that. There is not much benefit for me to having the bumped version, but that might just be me.